README.md

```diff
@@ -1,222 +1,211 @@
+
+Note that the SCSS partials are organized based on [SMACSS](http://smacss.com/) concepts into subfolders. Below is an example of what this structure might look like.
 - base - Set it and forget it. There should not be class selectors in these files.
   - _variables.scss
   - _colors.scss
   - _reset.scss (This is usually taken care of by a framework and not needed to add yourself.)
-  - _mixinss.sscss
+  - _mixins.sscss
   - _functions.scsss
-- vendor - Configurable variables provided by 3rd parties. No custom SCSS here. 
+- vendor - Configurable variables provided by 3rd parties. No custom SCSS here.
   - foundation-sites
     - _settings.scss
 - layout - Styles related to page layout. If using a grid system, there shouldn't be much here.
   - _page.scss
   - _print.scss
 - components - Separate components of the website, most custom code and overrides will be here.
   - _button.scss
-  - _forms.scss
-  - _input.scss
-  - _dialog.scss
-  - _toast.scss
-  - _menu.sscss
-  - _menu-item.scss
-  - ... etc....
+  - _card.scss
+  - _menu.scss
 - state - This is for components in different states, e.g. .button:hover. Often these changes are simple and can be included with the component itself, but sometimes it makees sense to put them here instead.
   - _menu-item.scss - might contain:
-     .menu-item.is-expanded {}
-     .menu-item.is-active {}
+  -  .menu-item.is-expanded {}
+  -  .menu-item.is-active {}
 - theme - Theme variations, e.g. if one section of the site is promotional and should have a different color theme.
   - _blogs.scss
   - _news.scss
-  
-### Best practices
-- In the styles-library folder which should be usable by other projects, wrap your scss in logical mixins so they do not generate CSS just by importing the file. This allows the end user to choose if they want to only use some parts of your styles. Foundation for Sites does this well, check their partials for examples.
-- Component specific variables should be at the top of every partial. Even if you are directly assigning `$input-margin: $global-margin`, if it is a value reused in many places in the file, consider creating a component specific variable. Putting them at the top of the file ensuress that they are easy to find.
-- No numbers, colors, etc, in the SCSS.
-Wrong:
-    ```
-    // All of these should be variables.
-    .my-component {
-      color: #ff0000;
-      font-size: 18px;
-      margin-bottom: 1rem;
-    }
-    ```
-Correct:
+
+## Development and contribution
+(@TODO this section should be present in the documentation. If somebody is reading this Readme, he already has access to repo and is probably meant to participate in the development...)
```

Comment:
Agreed.

Reply:
I suppose the links to styleguide provide in for example topbar, will be directed towards http://demo-intra.tieto.com:3000/ (Production though), not this repository, or not?

Reply:
This won't be an entry page of the Styleguide. I changed the config so that we can use separate files for GitHub repo Readme and styleguide Overview.